connaisseur/validators/cosign/cosign_validator.py

```diff
@@ -21,44 +22,110 @@
 class CosignValidator(ValidatorInterface):
     name: str
     trust_roots: list
+    vals: dict
```

Comment:
What does `vals` stand for? If it's just "values", I'd prefer a more telling name; otherwise, please explain in a comment.

Reply:
just noticed the change got lost